src/packages/@ncigdc/modern_components/Lolliplot/GeneLolliplot.relay.js

```diff
@@ -0,0 +1,72 @@
+/* @flow */
+/* eslint fp/no-class:0 */
+
+import React from 'react';
+import { graphql } from 'react-relay';
+import { compose, withPropsOnChange } from 'recompose';
+import { makeFilter } from '@ncigdc/utils/filters';
+import Query from '@ncigdc/modern_components/Query';
+
+export default (Component: Object) =>
+  compose(
+    withPropsOnChange(['geneId'], ({ geneId = 'ENSG00000134086' }) => {
```

Comment:
hmm I did think about that. what do you think of having a prop be something like `withDefault` so the caller doesn't need to know the internals in case they just want it to load on its own?